src/router.ts

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+import * as glob from 'glob'
+import logger from './util/logger';
+import * as path from 'path';
+//key index issue with types:so any
+const router:any = express.Router(); 
+
+const routes_list:Array<[string,string,Function]> = [];
```

Comment:
We should make this type narrower, something like
```suggestion
type Method = 'GET' | 'PUT' | 'POST' | 'PATCH' | 'DELETE' | <etc...>;
type Handler = (arg0: Request, arg1: Response) => (void | Promise<void>);
type Route = [string, Method, Handler];
const routes_list: Route[] = [];
```